### ISSUE TITLE
config object transforms are run twice

### ISSUE THREAD

**Issue by simonzack** [substack/node-browserify#1124]
Doing the following will run transforms twice, no matter what the transform is:

    var bundler = browserify('file.js', {
        transform: [to5ify]
    });

using `.transform` instead works.

**Comment by seansfkelley** [comment]
Seeing this as well. Here's a more complete repro:

```javascript
var browserify = require('browserify');
var through = require('through');

function transform1() {
  console.log('this log line appears once');
  return through();
};

function transform2() {
  console.log('this log line appears twice');
  return through();
};

browserify('./blank.js', {
  transform: [transform2]
})
.transform(transform1)
.bundle();
```

**Comment by substack** [comment]
All fixed in 9.0.2.
